build/build_apple_llm_demo.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+set -euo pipefail
+
+TOKENIZER="${1:-bpe}"
+ARTIFACTS_DIR_NAME="$2"
+
+APP_PATH="examples/demo-apps/apple_ios/LLaMA/LLaMA"
+
+if [[ "${TOKENIZER}" = "bpe" ]]; then
```

Comment:
This is very llm specific

Reply:
Yeah, I need this mainly to overwrite the default `ET_USE_TIKTOKEN=1` set throughout the app.  This might go away once we have a better generic iOS app at hand.

Reply:
cc: @shoumikhin